### PR TITLE
Scene2 BIN improvements.

### PR DESCRIPTION
**Pull request by zpl-zak**

### CODE REVIEW

File: components/scene2_bin/parser.hpp [hunk 1]
```diff
@@ -256,6 +297,7 @@ void DataFormatScene2BIN::readObject(std::ifstream &srcFile, Header* header, Obj
         case OBJECT_LIGHT_COLOUR:
         {
             read(srcFile, &object->mLightColour);
+            printf("ok?\n");
```
Comment (drummyfish) [inline components/scene2_bin/parser.hpp]: Forgot to delete?

Comment (zaklaus) [reply components/scene2_bin/parser.hpp]: Will do now, same with my small experiment I forgot to get rid of.

File: components/scene2_bin/parser.hpp [hunk 2]
```diff
@@ -274,7 +315,7 @@ void DataFormatScene2BIN::readObject(std::ifstream &srcFile, Header* header, Obj
         
         case OBJECT_LIGHT_SECTOR:
         {
-            read(srcFile, object->mLightSectors, header->mSize);
+            //read(srcFile, object->mLightSectors, header->mSize);
```
Comment (drummyfish) [inline components/scene2_bin/parser.hpp]: This is supposed to be commented out?

Comment (zaklaus) [reply components/scene2_bin/parser.hpp]: Temporarily, yes

**Review by drummyfish — APPROVED** [review]

**Pull request merged**